lib/posthog/feature_flags.rb

```diff
@@ -527,9 +539,11 @@ def _request(uri, request_object)
              EOFError,
              Net::HTTPBadResponse,
              Net::HTTPHeaderSyntaxError,
+             Net::ReadTimeout,
+             Net::WriteTimeout,
              Net::ProtocolError => e
         logger.debug("Unable to complete request to #{uri}")
-        throw e
+        raise e
```

Comment:
```suggestion
        raise
```

🙈 🤣

Reply:
These are great suggestions, seriously, thank you 😂